### ISSUE TITLE
Error when running the tutorial

### ISSUE THREAD

**Issue by timflutre** [SticsRPacks/SticsRPacks#17]
Hello,

I will be following the teaching session next week. I have a computer under GNU/Linux with Debian 12 "bookworm". My R versions is 4.2.2. My RStudio is 2023.06.1 build 524.

I installed SticsRPacks and I executed the following commands:
```
library(SticsRPacks)
learnr::run_tutorial("SticsRpacks", package="SticsRPacks")
```

I got the following error:
```
Quitting from lines 18-257 [setup] (SticsRpacks.Rmd)
Error:
! object 'is_unix' not found
Execution halted
```

I then updated SticsRPacks:
```
SticsRPacks_update()
learnr::run_tutorial("SticsRpacks", package="SticsRPacks")
```

Now I have the following error:
```
Quitting from lines 1425-1433 [gen_xml-1] (SticsRpacks.Rmd)
Error:
! 'mark' is not an exported object from 'namespace:markdown'
Backtrace:
 1. learnr::question(...)
 2. learnr::answer("gen_usms_xml2txt")
 3. learnr:::answer_new(...)
 4. learnr:::quiz_text(label)
Execution halted
```

What should I do? Thanks in advance!

**Comment by plecharpent** [comment]
Hi,
Could you verify if the markdown package is correctly installed and if it's version is 1.12

Try to remove SticsRPacks and reinstall it using
devtools::install_github("SticsRPacks/SticsRPacks@*release")

Regards

**Comment by timflutre** [comment]
Indeed Debian shipped an older version of  `markdown` (1.1). Now it is fine, thanks!
